### PR TITLE
Offseason Code for 2019 "Deep Space"

### PR DESCRIPTION
**Pull request by strimper13**
Additional code changes for offseason of the 2019 robot game

### CODE REVIEW

File: src/Vision/testpi.py [hunk 1]
```diff
@@ -300,7 +300,7 @@ def normalizeImage(pairMidpoint):
     fullPairMidpoint = (pairPointOrient / imageMidpoint)
     # prints after ewww math THIS IS THE CONTOUR MIDPOINT from...
     # Contour Midpoint Scale (-1 to 1)
-    return fullPairMidpoint
+    return fullPairMidpoint
```
Comment (Rohan-Sreenivasan) [inline src/Vision/testpi.py:303]: You have an extra space at the end of this

**Review by Rohan-Sreenivasan — COMMENTED** [review]
good code @strimper13

**Review by bot190 — APPROVED** [review]
This looks reasonable, and I want to close out last years code.

**Pull request merged**